cmd/dep/prune.go

```diff
@@ -68,6 +68,10 @@ func (cmd *pruneCommand) Run(ctx *dep.Ctx, args []string) error {
 		return errors.Wrap(err, "could not set up solver for input hashing")
 	}
 
+	if p.Lock == nil {
+		return fmt.Errorf("lock doesn't exists")
```

Comment:
Let's use this:

`Gopkg.lock must exist for prune to know what files are safe to remove.`